UI/src/app/services/backend-connector.service.ts

```diff
@@ -72,6 +74,30 @@ export class BackendConnectorService {
         //     });
     };
 
+   fileUpload(fileItem:File, extraData?:object):any{
+      var apiCreateEndpoint = 'http://localhost:3001/api/upload';
+      const formData: FormData = new FormData();
+
+      formData.append('fileItem', fileItem, fileItem.name);
+      if (extraData) {
+        for(let key in extraData){
+            // iterate and set other form data
+          formData.append(key, extraData[key])
+        }
+      }
+      var headers = new HttpHeaders();
+      headers.append('Content-Type', 'application/json');
+      headers.append('Accept', 'application/json');
+
+      return this.http.post(apiCreateEndpoint, formData, { headers: headers })
+                 .map((res : Response) => { return res; })
+                 .catch(error => Observable.throw(error));
+    };
+
+    public getDatabaseTables(){
+        var apiCreateEndpoint = 'http://localhost:3001/api/getTables';
```

Comment:
please use relative url, we have proxy in place.

Reply:
I changed to relative URL